### DISCUSSION TITLE
Star My repo and get a star bfor your repo

### DISCUSSION THREAD

**Discussion by pu-raihan** [drknzz/GitHub-Achievements#241] [Show and tell]
Hey guys,
Here's my project [CamBuzz](https://github.com/pu-raihan/CamBuzz) - A social media website and campus resource locator exclusively for Pondicherry University students.
Take a peek and star my repo. I'll do the same for you.
Happy coding!

**Comment by NagarAryan** [comment]
Hey @pu-raihan did my part.

**Reply by pu-raihan** [reply]
gimme the link to your repo

**Reply by NagarAryan** [reply]
Sure. https://github.com/NagarAryan/Flutter-App here you go

**Reply by pu-raihan** [reply]
👍🏻 done

**Reply by Mattral** [reply]
I starred yours please help mine too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by 0x4248** [comment]
> Hey guys, Here's my project [CamBuzz](https://github.com/pu-raihan/CamBuzz) - A social media website and campus resource locator exclusively for Pondicherry University students. Take a peek and star my repo. I'll do the same for you. Happy coding!
I will star yours and you star mine
https://github.com/Obsidian-tech-wiki/obsidian_tech_wiki

**Reply by pu-raihan** [reply]
Done

**Reply by 0x4248** [reply]
Thank you

**Reply by Mattral** [reply]
Hellp lewi,
I starred your please star mine too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by StefanShivarov** [comment]
Sure! Here's mine: 
https://github.com/StefanShivarov/modern-java-technologies-course-fmi

**Reply by Mattral** [reply]
Hello I starred yours, please star mine too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by pu-raihan** [reply]
Star mine and I'll star yours

**Comment by Mattral** [comment]
I starred yours please star mine too!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Reply by Mattral** [reply]
starred

**Comment by the1Riddle** [comment]
you will need 16 stars to get the star achievement on that.

**Reply by pu-raihan** [reply]
Yeah.. I got it... I'm trying for gold....star mine

**Reply by pu-raihan** [reply]
And put your repository link, I'll star yours

**Reply by the1Riddle** [reply]
here is the link to my project repo:
https://github.com/the1Riddle/AirBnB_clone

**Reply by the1Riddle** [reply]
i have added a star on yours

**Reply by pu-raihan** [reply]
@the1Riddle 👍🏻 done

**Reply by the1Riddle** [reply]
cool, thanks

**Reply by pu-raihan** [reply]
Since you're here, checkout my profile 😁😌

**Reply by the1Riddle** [reply]
what makes you think i haven't checked on your profile, Lol 😁

**Reply by the1Riddle** [reply]
can i offer a follow for a follow?

**Reply by pu-raihan** [reply]
Well well well why not

**Reply by the1Riddle** [reply]
i have done that already ^_^ ^-^

**Reply by Mattral** [reply]
lets follow each other!

**Reply by the1Riddle** [reply]
@Mattral am following you already, ha

**Comment by bettyc0der** [comment]
I starred your repo please star mine
https://github.com/salcedocharlotte/CIS261-Course-Project

**Reply by pu-raihan** [reply]
Done

**Reply by pu-raihan** [reply]
I didn't see your star...and check out my profile too and leave a star there too

**Comment by ganochenkodg** [comment]
@NagarAryan @pu-raihan done, pls star mine https://github.com/ganochenkodg/qdrant-operator
